src/node-utils.ts

```diff
@@ -322,14 +327,11 @@ function canContainDirective(node: ts.Node): boolean {
 /**
  * Returns line and column data for the given ts.Node or ts.Token,
  * for the given AST
- * @param  {ts.Token|TSNode} nodeOrToken the ts.Node or ts.Token
```

Comment:
in typescript there is no difference between node and token, both of types implements `ts.Node`

Reply:
Yeah this signature is a little odd, I agree. Would you mind tidying up the whole thing?

Reply:
ok i will do so :)